src/com/ms/silverking/cloud/dht/daemon/storage/NamespaceStore.java

```diff
@@ -743,8 +743,8 @@ private long newestVersion(DHTKey key) {
     // write lock must be held
     private Set<Waiter> checkPendingWaitFors(DHTKey key) {
         if (debugWaitFor) {
-            System.out.println("checkPendingWaitFors");
-            System.out.printf("pendingWaitFors.size() %d\n", pendingWaitFors.size());
+            Log.fineAsync("checkPendingWaitFors");
```

Comment:
debugWaitFor is off by default. Make Log.warningAsyncf so that the debugWaitFor flag has an effect